modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,297 @@
+
+| `fontsizes`                         | `fontsize`                       | Menu item                           | No behavioral changes.                                                                                                                                                                                                                                                                                       |
+
+| `formatpainter_blacklisted_formats` | `formatpainter_ignored_formats`  | Option                              | No behavioral changes.                                                                                                                                                                                                                                                                                       |
+
+| `getWhiteSpaceElements()`           | `getWhitespaceElements()`        | Function                            | This function is part of the `Schema` API. It was renamed as part of a general renaming that treats _Whitespace_ as a single word. The behavior of the function has not changed.                                                                                                                             |
+
+| `getShortEndedElements`             | `getVoidElements`                | API                                 | `special` and `void` are hard-coded elements defined by the HTML parsing specification. They can not be changed at run-time. The previous API name did not make it clear it was these, hard-coded, elements being called.                                                                                     |
+
+| `content`                           | `html`                           | Command                             | This `mceInsertClipboardContent` argument was renamed to better reflect what data is passed. NB: The `content` argument can no longer be used with `mceInsertClipboardContent`. If `content` is used, no data is passed.                                                                                      | see xref:changed-plugins-paste[Paste]
+
+| `linheight_formats`                 | `line_height_formats`            | Option                              | No behavioral changes.                                                                                                                                                                                                                                                                                       |
+
+| `default_link_target`               | `link_default_target`            |                                     | Changed for consistency with other `link` and `autolink` options. Their functionality, and the values they can take remain unchanged. This change applies to both `link` and `autolink` plugins.                                                                                                              |
```

Comment:
Is `their` and `they` correct here? It's a single option, but this makes it sound like it's multiple options.

Reply:
```suggestion
| `default_link_target`               | `link_default_target`            |                                     | Changed for consistency with other `link` and `autolink` options. The functionality, and the values the option can take remain unchanged. This change applies to both `link` and `autolink` plugins.                                                                                                              |
```